modules/version.lua

```diff
@@ -0,0 +1,21 @@
+local exports = {}
+
+-- Busted runs these modules scripts *outside* of Clink.
+-- So these Clink scripts have to work without any Clink APIs being available.
+clink = clink or {}
+
+local clink_version_encoded = clink.version_encoded or 0
+
+exports.supports_display_filter_description = function ()
```

Comment:
Does this (and the other exports as well) needs to be a function? Seems like it can be a plain variable which will be evaluated at script execution time when clink-injected variables are already available

Reply:
I see pros and cons in both approaches.  E.g. I reasoned that a function is more flexible for the long term, and would avoid a situation where someday some version checks can be variables but some must be functions.  Inconsistency would make it a fragile interface.

I think the execution time optimization there is so small that I would favor instead choosing based on semantic and maintenance considerations, i.e. the ability to prevent future bugs.

But the cons of that approach are not great enough for me to want to spend time on that conversation, so I'll make that change.

Reply:
I would like to request some room for stylistic and philosophical differences of opinion.  It can become frustrating if stylistic changes keep requiring verification passes.  That's a hidden cost that I would like to keep relatively small.